src/index.js

```diff
@@ -1,51 +1,56 @@
 import './style.css';
 
-const score = document.querySelector('.score');
-const formDiv = document.querySelector('.form-div');
+const API_KEY = 's5I1Iezs2fp5U1vv55v4';
+const URI = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${API_KEY}/scores`;
+const resultUl = document.querySelector('.result-ul');
+const displayScores = (list, res) => {
+  const { result } = res;
+  const scores = [];
+  for (let i = 0; i < result.length; i += 1) {
+    scores.push([result[i].user, result[i].score]);
+  }
+  list.innerHTML = '';
+  scores.forEach((score) => {
+    list.innerHTML += `<li>${score[0]}: ${score[1]}</li>`;
+  });
+  if (scores.length > 5) { list.classList.add('scroll'); }
+};
+const fetchData = async () => {
+  await fetch(URI, {
+    method: 'GET',
+  })
+    .then((response) => response.json())
+    .then((result) => displayScores(resultUl, result));
+};
```

Comment:
[OPTIONAL]

- Although you have a nice organized code, I think you can modularize the app by importing functions from another file and also separating API call functions from DOM rendering. This would make your file more readable, manageable, and structured. :pray:

Reply:
✔